kuma/wiki/migrations/0030_add_page_creators_group.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.core.management.sql import emit_post_migrate_signal
+
+
+def create_page_creators_group(apps, schema_editor):
+    # Ensure wiki.add_document permission is created in a fresh database
+    # https://code.djangoproject.com/ticket/23422#comment:20
+    db_alias = schema_editor.connection.alias
+    emit_post_migrate_signal(2, False, 'default', db_alias)
```

Comment:
I'm sorry, that wasn't a useful answer.  Here's a better one:

https://code.djangoproject.com/ticket/23422 is the Django bug that reports using ``Permission`` objects in data migrations is difficult because they are not created until a ``post_migrate`` signal is emitted, at the end of migrations.  This isn't a problem if you've already ran the other 29 migrations, and are just running this one.  You run into problems when running migrations on an empty database (like we do in TravisCI), when the migration that refers to permissions will fail.

I ran into this [14 months ago](https://code.djangoproject.com/ticket/23422#comment:6), and [shipped a solution](https://github.com/mdn/browsercompat/blob/master/mdn/migrations/0002_data_add_group.py) based on manually creating the permission objects. However, the [later solution](https://code.djangoproject.com/ticket/23422#comment:20) in the bug report is cleaner, and is used here.

The good news is that [someone is working on a solution](https://code.djangoproject.com/ticket/23422#comment:26) that may be included in Django 1.10. 

The alternate to all this is my previous strategy: don't create groups and assign permissions in a data migration. Instead, do it in the Django admin on the running server.  The MDN development instructions include creating a superuser for development, which will get all the permissions by default, so it's not a requirement for 95% of developers either.

But, we have the migration, so I think we should use it, even if it is ugly.